Cargo.toml

```diff
@@ -47,17 +47,23 @@ name = "_pydantic_core"
 crate-type = ["cdylib", "rlib"]
 
 [features]
-# must be disabled for cargo bench
-extension-module = ["pyo3/extension-module"]
```

Comment:
Looks like this can be safely removed (well, moved into the maturin config) thanks to the changes from this (recent) PR: https://github.com/PyO3/maturin/pull/1479

Specifically, see this comment indicating that the intent was to make it so `cargo test` etc. can be used without flags: https://github.com/PyO3/maturin/pull/1479#issuecomment-1427474853